src/test/scala/org/zalando/spearheads/innkeeper/services/RoutesServiceSpec.scala

```diff
@@ -26,13 +27,14 @@ import scala.language.postfixOps
  */
 class RoutesServiceSpec extends FunSpec with Matchers with MockFactory with ScalaFutures {
 
+  override implicit val patienceConfig = PatienceConfig(timeout = Span(1000, Seconds))
```

Comment:
1000 seconds? Isn't that a bit too patient? If a test runs > 15 min, something is probably broken